lib/jekyll/commands/new.rb

```diff
@@ -114,6 +145,11 @@ def site_template
         def scaffold_path
           "_posts/0000-00-00-welcome-to-jekyll.markdown.erb"
         end
+
+        def user_email
```

Comment:
This is [similar to `Jekyll::ThemeBuilder.user_email`](https://github.com/jekyll/jekyll/blob/f4c3c5afd6f7d051934eaf67d60ebe413b7214a8/lib/jekyll/theme_builder.rb#L90-L92). Is there any way to DRY up?